creating/help.adoc

```diff
@@ -20,27 +20,31 @@ The help file must be located in the images as /help.1 and in 'man' format.
 
 The following headings are strongly encouraged in the help file for an image.
 
+===== NAME
+Image name with short description.
+
 ===== DESCRIPTION
-Describe in greater detail the role or purpose of the image.  This can include more specifics about the
-packages that make up the image.  You can also describe whether image is meant to be interactive
-or more service oriented.
+Describe how image is used (user app, service, base image, builder image, etc.), the services or features it provides, and environment it is intended to run in (stand-alone docker, atomic super-privileged, oc multi-container app, etc.).
+
+===== USAGE
+Describe how to run the image as a container and what factors might influence the behavior of the image itself. Provide specific command lines that are appropriate for how the container should be run.
 
-==== USAGE
-Describe how to run the image as a container and what factors might influence the behaviour of the image
-itself.
+===== ENVIROMENT VARIABLES
+Explain all environment variables available to run the image in different ways without the need of rebuilding the image.
 
-==== HISTORY
+===== HISTORY
 Similar to a Changelog of sorts which can be as detailed as the maintainer desires.
 
 ==== Optional headings
 
 Use the following sections in your help file when applicable.
 
-===== Labels
-Describe the labels you use and why when applicable.  Specifically focus any action-oriented labels like RUN, INSTALL, and
-UNINSTALL.
+===== LABELS
+Describe LABEL settings (from the Dockerfile that created the image) that contains pertinent information.
```

Comment:
One more thing, a link to the label repo could be useful: https://github.com/projectatomic/ContainerApplicationGenericLabels/tree/master/vendor/redhat